src/session/__tests__/sessionReducer-test.js

```diff
@@ -38,4 +38,39 @@ describe('sessionReducer', () => {
     const newState = sessionReducer(baseState, eg.action.login_success);
     expect(newState).toEqual({ ...baseState, needsInitialFetch: true });
   });
+
+  test('DEAD_QUEUE', () => {
+    const newState = sessionReducer(baseState, eg.action.dead_queue);
+    expect(newState).toEqual({ ...baseState, needsInitialFetch: true });
+  });
+
+  test('LOGOUT', () => {
+    const newState = sessionReducer(baseState, eg.action.logout);
+    expect(newState).toEqual(baseState);
```

Comment:
It's not very informative to say that this has no effect when starting from this state :wink: . Better to start from a state where it does do something. See the `CANCEL_EDIT_MESSAGE` test for an example.

Reply:
Updated 👍